modules/installation-osp-kuryr-settings-installing.adoc

```diff
@@ -0,0 +1,111 @@
+----
+<1> For `<installation_directory>`, specify the name of the directory that
+contains the `install-config.yaml` file for your cluster.
+
+. Create a file that is named `cluster-network-03-config.yml` in the
+`<installation_directory>/manifests/` directory:
++
+[source,terminal]
+----
+$ touch <installation_directory>/manifests/cluster-network-03-config.yml <1>
+----
+<1> For `<installation_directory>`, specify the directory name that contains the
+`manifests/` directory for your cluster.
++
+After creating the file, several network configuration files are in the
+`manifests/` directory, as shown:
++
+[source,terminal]
+----
+$ ls <installation_directory>/manifests/cluster-network-*
+----
++
+.Example output
+[source,terminal]
+----
+cluster-network-01-crd.yml
+cluster-network-02-config.yml
+cluster-network-03-config.yml
+----
+
+. Open the `cluster-network-03-config.yml` file in an editor and enter a custom
+resource (CR) that describes the Cluster Network Operator configuration that you want:
++
+[source,terminal]
+----
+$ oc edit networks.operator.openshift.io cluster
+----
+
+. Edit the settings to meet your requirements. The following file is provided as an example:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  serviceNetwork:
+  - 172.30.0.0/16
+  defaultNetwork:
+    type: Kuryr
+    kuryrConfig:
+      enablePortPoolsPrepopulation: false <1>
+      poolMinPorts: 1 <2>
+      poolBatchPorts: 3 <3>
+      poolMaxPorts: 5 <4>
+      openstackServiceNetwork: 172.30.0.0/15 <5>
+----
+<1> Set `enablePortPoolsPrepopulation` to `true` to make Kuryr
+create new Neutron ports when a namespace is created or a new node is added to
+the cluster. This setting raises the Neutron ports quota, but can reduce the time that is required to spawn pods. The default value is `false`.
+<2> Kuryr creates new ports for a pool when the number of free ones in that
+pool is lower than `poolMinPorts`. The default value is `1`.
+<3> `poolBatchPorts` controls the number of new ports that are created when the number of
+free ports is lower than `poolMinPorts`. The default value is `3`.
+<4> If the number of free ports in a pool is higher than `poolMaxPorts`, Kuryr
+deletes them until the number matches the value of `poolMaxPorts`. Setting `0`
+disables this upper bound, preventing pools from shrinking. The default value is `0`.
+<5> The `openStackServiceNetwork` parameter defines the CIDR range of the network from which
+IP addresses are allocated to {rh-openstack} Octavia's LoadBalancers.
++
+If this parameter is used with the Amphora driver, Octavia takes two IPs from this network for each loadbalancer:
+one for OpenShift, and the other for VRRP connections. Because these IP addresses are managed by the IPAMs of OpenShift and Neutron respectively, they must come from different pools.
```

Comment:
```suggestion
If this parameter is used with the Amphora driver, Octavia takes two IP addresses from this network for each load balancer: one for OpenShift, and the other for VRRP connections. Because these IP addresses are managed by {product-title} and Neutron respectively, they must come from different pools.
```